src/main/java/utils/variables/SessionVariables.java

```diff
@@ -6,6 +6,7 @@
     VAR_USER_ID("id"),
     VAR_POST_ID("id"),
     VAR_COMMENT_ID("id"),
+    VAR_ALBUM_ID("id"),
```

Comment:
Why you have the same key for different id's? How serenity will know to retrieve the id you want(for album/user/etc?